hiera/hiera.go

```diff
@@ -185,7 +239,12 @@ func LookupAndRender(c api.Session, opts *CommandOptions, args []string, out io.
 		explainer = explain.NewExplainer(opts.ExplainOptions, opts.ExplainOptions && !opts.ExplainData)
 	}
 
-	found := Lookup2(c.Invocation(createScope(c, opts), explainer), args, tp, dv, nil, nil, options, nil)
+	var found dgo.Value
+	if opts.MultiLookup {
+		found = MultiLookup(c.Invocation(createScope(c, opts), explainer), args, tp, dv, nil, nil, options, nil)
```

Comment:
An assertion could be made here that if `tp` is provided, it should be a `dgo.StructMapType`.